owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/PassCodeActivityTest.kt

```diff
@@ -95,6 +96,63 @@ class PassCodeActivityTest {
         PreferenceManager.getDefaultSharedPreferences(context).edit().clear().commit()
     }
 
+    @Test
+    fun passcodeCheckNotLockedView() {
+        //Open Activity in passcode check mode
+        openPasscodeActivity(PassCodeActivity.ACTION_CHECK)
+
+        with(R.id.header) {
+            isDisplayed(true)
+            withText(R.string.pass_code_enter_pass_code)
+        }
+        with(R.id.explanation) {
+            isDisplayed(false)
+        }
+        // Check if required amount of input fields are actually displayed
+        with(R.id.passCodeTxtLayout) {
+            isDisplayed(true)
+            withChildCountAndId(passCodeViewModel.getNumberOfPassCodeDigits(), R.id.passCodeEditText)
+        }
+
+        with(R.id.cancel) {
+            isDisplayed(false)
+        }
+
+        with(R.id.lock_time) {
+            isDisplayed(false)
+        }
```

Comment:
I would avoid `with` clauses for one statement asserts.
```suggestion
        R.id.cancel.isDisplayed(false)

        R.id.lock_time.isDisplayed(false)
```

Reply:
👍